action_server/src/robocorp/action_server/_server_expose.py

```diff
@@ -53,12 +54,14 @@ def read_expose_session_json(datadir: str) -> None | ExposeSessionJson:
     return session_json
 
 
-def write_expose_session_json(datadir: str, expose_session: str, url: str) -> None:
+def write_expose_session_json(
+    datadir: str, expose_session_json=ExposeSessionJson
```

Comment:
I think you meant to write:
`expose_session_json:ExposeSessionJson`
instead of:
`expose_session_json=ExposeSessionJson`

i.e.: set the type, not the value

Reply:
Just saw that it's already merged. I'll fix it here then.